### ISSUE TITLE
Do not route through pointless loops

### ISSUE THREAD

**Issue by Dimitar5555** [Zaczero/osm-relatify#63]
The easiest way to go from PTv1 to PTv2 relations, is to duplicate the the PTv1 relation for every route variation/direction and then edit them in Relatify.Since PTv1 usually contains all directions in one relation, most routes contain roads that can loop around. Relatify should be able to see that the bus does not stop on any stop on that road and ignore it when trying to "navigate" (which should make them appear in the "Unused ways" error).

The route in the screenshots below goes west and then south.

![image](https://github.com/Zaczero/osm-relatify/assets/19364673/190e102d-8711-45b9-bf0e-19dfba09bcc9)
![image](https://github.com/Zaczero/osm-relatify/assets/19364673/a1f052be-a3b8-406c-98d7-42ed7834d819)

**Comment by Zaczero** [comment]
This is a rather complex problem to address, and I believe it would require a significant amount of time and effort. At the moment, I don't think it's justified in terms of the time it would save for the user (compared to the development time).

**Comment by Dimitar5555** [comment]
Then would it be possible to have an information panel that says that there are X loopbacks (or places where the route passes two or more times) and links to their location?

**Comment by Zaczero** [comment]
Yes, that's easier and more feasible :) This could be done as a part of the config panel feature. So people can turn this feature on as they wish. I don't have any time ETA for the implementation, I do not currently focus on new Relatify features - most of my time goes towards OpenStreetMap NextGen development.
